source/administration/changelog.md

```diff
@@ -37,9 +35,10 @@ Added significant improvements for large organizations running on a single serve
 #### Favorite Channels
 - Added the ability to select Favorite Channels that appear at the top of the channels sidebar.
 
-#### Video and Audio Calling (Beta)
+#### Video and Audio Calling (early-preview)
```

Comment:
In my mind "Beta" is fine. We don't use "early-preview" anywhere else. But I'm only 1/5